src/config.prod.json

```diff
@@ -0,0 +1,5 @@
+{
+	"wordpress.com": {
+		"auth": "proxy"
+	}
+}
```

Comment:
As discussed IRL, I think we should use OAuth here instead.  It will require users to log in again, but it's a lot less hacky than the rest-proxy thing, and makes it a lot easier to debug API requests.